lib/ansible/modules/files/blockinfile.py

```diff
@@ -12,102 +12,95 @@
     default: present
   marker:
     description:
-      - The marker line template.
-        "{mark}" will be replaced with the values in marker_begin
-        (default="BEGIN") and marker_end (default="END").
+    - The marker line template.
+    - C({mark}) will be replaced with the values C(in marker_begin) (default="BEGIN") and C(marker_end) (default="END").
     default: '# {mark} ANSIBLE MANAGED BLOCK'
   block:
     description:
-      - The text to insert inside the marker lines.
-        If it's missing or an empty string,
-        the block will be removed as if C(state) were specified to C(absent).
+    - The text to insert inside the marker lines.
+    - If it is missing or an empty string, the block will be removed as if C(state) were specified to C(absent).
     aliases: [ content ]
     default: ''
   insertafter:
     description:
-      - If specified, the block will be inserted after the last match of
-        specified regular expression. A special value is available; C(EOF) for
-        inserting the block at the end of the file.  If specified regular
-        expression has no matches, C(EOF) will be used instead.
+    - If specified, the block will be inserted after the last match of specified regular expression.
+    - A special value is available; C(EOF) for inserting the block at the end of the file.
+    - If specified regular expression has no matches, C(EOF) will be used instead.
     default: EOF
     choices: [ EOF, '*regex*' ]
   insertbefore:
     description:
-      - If specified, the block will be inserted before the last match of
-        specified regular expression. A special value is available; C(BOF) for
-        inserting the block at the beginning of the file.  If specified regular
-        expression has no matches, the block will be inserted at the end of the
-        file.
+    - If specified, the block will be inserted before the last match of specified regular expression.
+    - A special value is available; C(BOF) for inserting the block at the beginning of the file.
+    - If specified regular expression has no matches, the block will be inserted at the end of the file.
     choices: [ BOF, '*regex*' ]
   create:
     description:
-      - Create a new file if it doesn't exist.
+    - Create a new file if it does not exist.
     type: bool
-    default: 'no'
+    default: no
   backup:
     description:
-      - Create a backup file including the timestamp information so you can
-        get the original file back if you somehow clobbered it incorrectly.
+    - Create a backup file including the timestamp information so you can
+      get the original file back if you somehow clobbered it incorrectly.
     type: bool
-    default: 'no'
+    default: no
   marker_begin:
     description:
-      - This will be inserted at {mark} in the opening ansible block marker.
-    default: 'BEGIN'
-    version_added: "2.5"
+    - This will be inserted at {mark} in the opening ansible block marker.
```

Comment:
should {mark} (here and line 83) be in C( . . . )?

Reply:
Yes it should, these are values or pieces of values. Thanks !

Reply:
Fixed.